x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshots.java

```diff
@@ -150,6 +150,7 @@
     private final SetOnce<BlobStoreCacheService> blobStoreCacheService = new SetOnce<>();
     private final SetOnce<CacheService> cacheService = new SetOnce<>();
     private final SetOnce<ThreadPool> threadPool = new SetOnce<>();
+    private final SetOnce<ClusterService> clusterService = new SetOnce<>();
```

Comment:
This is unused?

Reply:
It is, I removed it in ae5895ce811addbc5094133031dabec540882c02